lib/matplotlib/tests/test_axes.py

```diff
@@ -797,6 +797,23 @@ def test_polar_rlim_zero():
     assert ax.get_ylim()[0] == 0
 
 
+@image_comparison(['aitoff_proj'], extensions=["png"], remove_text=True)
+def test_aitoff_proj():
+    """
+    Test aitoff projection ref.:
+    https://github.com/matplotlib/matplotlib/pull/14451
+    """
+    np.random.seed(42)
+    ra_random = np.random.rand(20) * 2 * np.pi
+    dec_random = (np.random.rand(20) * 180.0 - 90.0) * 2 * np.pi
+
+    fig, ax = plt.subplots(figsize=(8, 4.2),
+                           subplot_kw=dict(projection="aitoff"))
+    ax.tick_params(left=False, bottom=False,
```

Comment:
Why remove the ticks?

Reply:
Because they aren't relevant for what's being tested. Would you like me to include them?

Reply:
If we add them then the image is also testing the tick layout, and will be a good catch of any changes in ticking changes for this projection.

Reply:
aitoff projection doesn't have ticks, and the labels are excluded by the `remove_text=True` option in the @image_comparison decorator (I cannot get the matching freetype to install on my system). But I went to put in a `grid` now, which would test the tick locations.